services/business/istio_validations_test.go

```diff
@@ -371,3 +379,28 @@ func fakeCombinedServices(services []string) *kubernetes.ServiceList {
 	}
 	return &serviceList
 }
+
+func fakePods() *v1.PodList {
+	return &v1.PodList{
+		Items: []v1.Pod{
+			v1.Pod{
+				ObjectMeta: meta_v1.ObjectMeta{
+					Name: "reviews-12345-hello",
+					Labels: map[string]string{
```

Comment:
Commented out, just to be sure that the mock pod has the correct type used by the real pod, map[string]string vs map[string]interface{}.

It just to double check this.

Reply:
Yes, k8s uses map[string]string for labels whereas IstioObject uses map[string]interface{}. I had to deal with that before and had to add those castings instead.